lib/yoti/activity_details.rb

```diff
@@ -96,6 +103,7 @@ def initialize(receipt, decrypted_profile = nil, decrypted_application_profile =
       @timestamp = receipt['timestamp'] ? Time.parse(receipt['timestamp']) : nil
       @extended_user_profile = process_decrypted_profile(decrypted_profile)
       @extended_application_profile = process_decrypted_profile(decrypted_application_profile)
+      @extra_data = Share::ExtraData(extra_data) if extra_data
```

Comment:
Could we add coverage for `attr_reader :extra_data` in `activity_details_spec.rb`?